packages/webpack-configs/src/utils.ts

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function isObject(value: any): value is Record<string, unknown> {
```

Comment:
not sure why this implementation is different than the one from lodash which is probably the most popular implementation. 

you don't need to validate the array for your use cases

Reply:
I copied the implementation from Orbit which has been working fine for us: https://github.com/gsoft-inc/sg-orbit/blob/master/packages/components/src/shared/src/assertions.ts#L41